site/pages/create-disruption.tsx

```diff
@@ -10,58 +11,192 @@ interface CreateDisruptionProps {
+                                <legend className="govuk-fieldset__legend govuk-!-padding-top-2">
+                                    <h3 className="govuk-heading-s govuk-!-margin-bottom-0" id="summary">
+                                        Summary
+                                    </h3>
+                                </legend>
+                                <input className="govuk-input w-3/4" id="summary" name="summary" type="text" />
+                            </div>
+                            <div className="govuk-form-group">
+                                <legend className="govuk-fieldset__legend govuk-!-padding-top-2">
+                                    <h3 className="govuk-heading-s govuk-!-margin-bottom-0" id="description">
+                                        Description
+                                    </h3>
+                                </legend>
+                                <textarea
+                                    className="govuk-textarea w-3/4"
+                                    id="description"
+                                    name="description"
+                                    rows={5}
                                 />
-                                <label className="govuk-label govuk-radios__label" htmlFor="disruption-repeats">
-                                    Yes
-                                </label>
                             </div>
-                            <div className="govuk-radios__item">
+                            <div className="govuk-form-group">
+                                <legend className="govuk-fieldset__legend govuk-!-padding-top-2">
+                                    <h3 className="govuk-heading-s govuk-!-margin-bottom-0" id="associated-link">
+                                        Associated Link
+                                    </h3>
+                                </legend>
                                 <input
-                                    className="govuk-radios__input"
-                                    id="disruption-does-not-repeat"
-                                    name="disruptionRepeats"
-                                    type="radio"
-                                    value="no"
+                                    className="govuk-input w-3/4"
+                                    id="associated-link"
+                                    name="associatedLink"
+                                    type="text"
                                 />
-                                <label className="govuk-label govuk-radios__label" htmlFor="disruption-does-not-repeat">
-                                    No
-                                </label>
+                            </div>
+                            <div className="govuk-form-group">
+                                <legend className="govuk-fieldset__legend govuk-!-padding-top-2">
```

Comment:
This should be a label rather than a legend

Reply:
done